src/libraries/System.Private.CoreLib/src/System/IO/Path.Windows.cs

```diff
@@ -183,10 +183,12 @@ internal static void GetTempPath(ref ValueStringBuilder builder)
             static uint GetTempPathW(int bufferLen, ref char buffer)
             {
                 delegate* unmanaged<int, char*, uint> func = s_GetTempPathWFunc;
+#pragma warning disable IDE0074 // Use compound assignment
                 if (func == null)
                 {
                     func = s_GetTempPathWFunc = GetGetTempPathWFunc();
                 }
+#pragma warning restore IDE0074
```

Comment:
Can we make sure there's an open issue in dotnet/roslyn for this?  The analyzer shouldn't fire for function pointers unless ??= can be used with function pointers.

Reply:
I need to check this one again, but I think the reason I didn't want to use the fix was for clarity around the call to `GetGetTempPathWFunc()`. I'll try using `??=` here and see if it works for function pointers.

Reply:
> I'll try using ??= here and see if it works for function pointers.

I don't believe it does.

Reply:
You are correct. It does not work for function pointers. I'll open an issue.